src/components/CoinPile/CoinPile.js

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components'
+import coin from '../../assets/qcoin.png'
+
+export default function CoinPile({ coins }) {
+  return (
+    <StyledCoinPile>
+      {[...new Array(coins)].map(() => (
+        <StyledCoin src={coin} />
```

Comment:
Idk but do we need here a key prop?

Reply:
Good catch! As one coin is like the others a key is not really needed, but react will complain anyway.